Providers/Profile.swift

```diff
@@ -444,7 +444,8 @@ open class BrowserProfile: Profile {
     }
 
     public func sendItems(_ items: [ShareItem], toClients clients: [RemoteClient]) -> Deferred<Maybe<SyncStatus>> {
-        let id = DeviceInfo.clientIdentifier(self.prefs)
+        let scratchpadPrefs = self.prefs.branch("sync.scratchpad")
+        let id = scratchpadPrefs.stringForKey("clientGUID")!
```

Comment:
Code style is to avoid force unwraps, use:`if let id = scratchpadPrefs.stringForKey("clientGUID") {}` 
or `guard let id = scratchpadPrefs.stringForKey("clientGUID") else { return }`

Optionally, nil case handling can be roughly
` } else { Sentry.shared.sendWithStacktrace(message: "a thing that should never be nil, is nil!", ...)`

The linter will catch these cases also.